php/php.editor/src/org/netbeans/modules/php/editor/completion/DocRenderer.java

```diff
@@ -608,9 +608,12 @@ private String composeType(Collection<TypeResolver> types, Type.Kind typeKind) {
                 if (type.isResolved()) {
                     QualifiedName typeName = type.getTypeName(true);
                     if (typeName != null) {
-                        if (sb.length() > 0
-                                && (typeKind == Type.Kind.UNION || typeKind == Type.Kind.INTERSECTION)) {
-                            sb.append(" ").append(typeKind.getSign()).append(" "); // NOI18N
+                        if (sb.length() > 0) {
+                                if (typeKind == Type.Kind.INTERSECTION) {
+                                    sb.append(" ").append(typeKind.getSign()).append(" "); // NOI18N
+                                } else {
+                                    sb.append(" ").append(Type.Kind.UNION.getSign()).append(" "); // NOI18N
```

Comment:
It would be nice if you can write the reason as a comment here.

Reply:
Personally, I would rather like to see `else if (UNION). {...` here.

Reply:
because of the same code... maybe...
```java
if (typeKind == Type.Kind.INTERSECTION || typeKind == Type.Kind.UNION) {
   sb.append(...
} else {
    // the reason
   sb.append(...
}
```

Reply:
My assumption is that if more than one type is specified and it is clearly not an intersection, it is always a union. Or is it not so?
Similar logic in NavigatorScanner:
https://github.com/apache/netbeans/blob/0bcc927866938e6c7f57e1bfe8f852e983b3072f/php/php.editor/src/org/netbeans/modules/php/editor/csl/NavigatorScanner.java#L425-L446

Reply:
Added a comment with a reason to the code.

Reply:
> it is always a union. Or is it not so?

Well... not so. PHP 8.2 has DNF type. Please wait a while because I'll improve it. (I have not finished implementing it yet.)

I would do the above because we can see that the `else` part is not a case of normal union type. Your change is also no problem, of course. Thanks.